docs/dev/install.rst

```diff
@@ -86,15 +86,28 @@ Installation
 
    - ``SITE_URL`` should be set to the URL you will use to connect to your
      local development site. Some people prefer to use
-     ``http://127.0.0.1:8000`` instead of ``localhost``.
+     ``http://127.0.0.1:8000`` instead of ``localhost``. However, should you
+     decide to change the ``SITE_URL``, you also need to
+     [request](https://developer.mozilla.org/en-US/docs/Mozilla/Tech/Firefox_Accounts/Introduction)
```

Comment:
Maybe move the link out of the paragraph for easier reading and consistency with the rest of the document? Check out the links section under "The site should be available at http://localhost:8000."